addons/l10n_hk/tests/test_l10n_hk_emv_qr.py

```diff
@@ -79,10 +84,7 @@ def l10n_hk_test_emv_qr_vals(self):
             structured_communication=self.emv_qr_invoice.payment_reference,
         )
 
-        # Check crc16 value
-        crc16 = emv_qr_vals[-1]
-        self.assertEqual(crc16, '3CA6')
-
         # Check the whole qr code string
         qr_code_string = ''.join(emv_qr_vals)
-        self.assertEqual(qr_code_string, '00020101021226330012hk.com.hkicl0313+852-678912345204000053033445405115.05802HK5914company_1_data6002HK63043CA6')
+        self.assertEqual(qr_code_string,
+            '00020101021226330012hk.com.hkicl0313+852-678912345204000053033445405100.05802HK5914company_1_data6002HK62150511INVTEST0001630442D7')
```

Comment:
Not important, but I think it is clearer to put all the params on its own line if it's too long to fit in a single one.
```suggestion
        self.assertEqual(
            qr_code_string,
            '00020101021226330012hk.com.hkicl0313+852-678912345204000053033445405100.05802HK5914company_1_data6002HK62150511INVTEST0001630442D7'
        )
```